src/reducers/profile-view.js

```diff
@@ -419,6 +420,21 @@ function isCallNodeContextMenuVisible(state: boolean = false, action: Action) {
   }
 }
 
+function profileSharingStatus(
+  state: ProfileSharingStatus = {
+    sharedWithUrls: false,
+    sharedWithoutUrls: false,
+  },
+  action: Action
+) {
```

Comment:
We didn't do that very well in this file but started doing it in other file: please type the return type of the reducer to. It should be the state, so `ProfileSharingStatus` in this case.